From b0a251b532e1ecc28d912ce556fd7f302d54901e Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Wed, 1 Dec 2021 15:43:34 +0800
Subject: [PATCH 8/9] no warn if we find content in the deployment's /var

Since commit [edf7477e deploy: Warn if we find content in the
deployment's /var] applied, warns in LAT but harmless

Revert the commit for no warn

Upstream-Status: Inappropriate [LAT specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Rebased on 20230607 for ostree 2023.3.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 src/libostree/ostree-sysroot-deploy.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/libostree/ostree-sysroot-deploy.c b/src/libostree/ostree-sysroot-deploy.c
index 158a4a5..684b8e2 100644
--- a/src/libostree/ostree-sysroot-deploy.c
+++ b/src/libostree/ostree-sysroot-deploy.c
@@ -3179,9 +3179,6 @@ sysroot_initialize_deployment (OstreeSysroot *self, const char *osname, const ch
   if (!prepare_deployment_etc (self, repo, new_deployment, deployment_dfd, cancellable, error))
     return FALSE;
 
-  if (!lint_deployment_fs (self, new_deployment, deployment_dfd, cancellable, error))
-    return FALSE;
-
   ot_transfer_out_value (out_new_deployment, &new_deployment);
   return TRUE;
 }
-- 
2.27.0

